src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+	private Person person;
+	private Tag tag;
+	private String tagSign;
+	private static ArrayList<String> taggingList = new ArrayList<String>();
```

Comment:
It would be advisable to make an ArrayList of Tagging objects instead of String. To print the object you can call its toString method.